### ISSUE TITLE
README.md wrong instructions for authorizing "t" app

### ISSUE THREAD

**Issue by reikred** [sferik/t#460]
So many problems...

1. the instructions in README.md specifies the link https://apps.twitter.com/app/new , which is not valid.

2. it appears from  https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api that one cannot get even the lowest level of API access (so-called "essential access") without having a phone-verified account. Not even read-only access, it seems.

3. the "t authorize" command is likewise useless and opens a chrome browser to URL https://developer.twitter.com/apps, but the instructions do not match the reality of that link.

In conclusion, the "t" program and documentation is outdated relative to twitter reality. And "t" may not work at all unless you are willing to give up your privacy with a phone number.

I'd love to be proven wrong but right now I'm stuck with the above problems.

**Comment by tsummerall** [comment]
I can't get _t authorize_ to work even with an account with a phone number.

**Comment by MHBraun** [comment]
Neither 
t authorize
Nor
t authorize --display-uri
do work as Twitter changed their platform.

**Comment by HacDan** [comment]
Any further progress on this?
